src/core/qgsjsonutils.h

```diff
@@ -345,6 +345,19 @@ class CORE_EXPORT QgsJsonUtils
      */
     static json jsonFromVariant( const QVariant &v ) SIP_SKIP;
 
+    /**
+     * Converts JSON QString \a jsonString to a QVariant, in case of parsing error an invalid QVariant is returned.
```

Comment:
```suggestion
     * Converts JSON \a jsonString to a QVariant, in case of parsing error an invalid QVariant is returned.
```

I think the types in the two docstring here and below are mixed up. But I'd just skip them both, they are trivial to see from the signature itself.